app/models/education_benefits_claim.rb

```diff
@@ -89,7 +93,7 @@ def selected_benefits
 
     if is_1990?
       benefits = parsed_form.slice(*APPLICATION_TYPES)
-    elsif is_1990e?
+    elsif is_1990e? || is_5490?
```

Comment:
that method is used in create_education_benefits_submission so it would be easier to keep it in the model